complex/ft_comp.c

```diff
@@ -50,6 +50,24 @@ static size_t comp_entry_size[] = {
 };
 */
 
+static int ft_open_cntrs(void)
+{
+	int ret;
+
+	opts.comp_method = test_info.cq_wait_obj;
```

Comment:
Coverity caught this as well.  Something isn't right here.  I'll try to fix-up the coverity errors.  @aingerson is mostly unavailable this week.